src/Automation/SnapshotCommand.cs

```diff
@@ -86,5 +74,25 @@ public static ScanResults Execute(Config config, IOutputFileHelper outputFileHel
                 } // using
             });
         }
-    }
-}
+
+        private static (string A11yTest, string Sarif) WriteOutputFiles(OutputFileFormat outputFileFormat, IOutputFileHelper outputFileHelper, ElementContext elementContext)
+        {
+            string a11yTestOutputFile = null;
+
+            if (outputFileFormat.HasFlag(OutputFileFormat.A11yTest))
+            {
+                ScreenShotAction.CaptureScreenShot(elementContext.Id);
+
+                a11yTestOutputFile = outputFileHelper.GetNewA11yTestFilePath();
+                SaveAction.SaveSnapshotZip(a11yTestOutputFile, elementContext.Id, elementContext.Element.UniqueId, A11yFileMode.Test);
+            }
+
+#if NOT_CURRENTLY_SUPPORTED
+                                if (locationHelper.IsSarifExtension())
+                                    // SaveAction.SaveSarifFile(outputFileHelper.GetNewSarifFilePath(), ec2.Id, !locationHelper.IsAllOption());
+#endif
```

Comment:
Can we just remove lines 90-93 for now? They're using the old classes and old assumptions, and could easily be replaced with a comment along the lines of:

// Future: If needed, generate Sarif output via SaveAction.SaveSarifFIle

Reply:
I'm keeping it because having that code will make it easier once we support sarif 2.1.